### PR TITLE
Add in several date functions, and other string functions

### PR DESCRIPTION
**Pull request by jsnoble**
added several functions to dates, strings and added toNumber

### CODE REVIEW

File: packages/data-mate/src/function-configs/date/isToday.ts [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+        },
+        {
+            args: {},
+            config: { version: 1, fields: { testField: { type: FieldType.String } } },
+            field: 'testField',
+            input: '2020-05-09T10:00:00.000Z',
+            output: null
+        },
+    ],
+    description: 'Determines if the given date is on the same day (server-time)',
```
Comment (peterdemartini) [inline packages/data-mate/src/function-configs/date/isToday.ts:31]: "server time" is not correct, servers can be configured with a time zone (even though we don't), it is better to be clear that it is UTC

**Review by peterdemartini — CHANGES_REQUESTED** [review]
One small change then lets merge this

File: packages/data-mate/src/function-configs/date/isTomorrow.ts [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+        {
+            args: {},
+            config: { version: 1, fields: { testField: { type: FieldType.String } } },
+            field: 'testField',
+            input: tomorrowDate,
+            output: tomorrowDate,
+            description: 'represents day after current time'
+        },
+    ],
+    description: 'Determines if the given date is on the next day (server-time)',
```
Comment (ciorg) [inline packages/data-mate/src/function-configs/date/isTomorrow.ts:35]: should be UTC

**Review by peterdemartini — APPROVED** [review]
LGTM

**Review by ciorg — CHANGES_REQUESTED** [review]

File: packages/data-mate/src/function-configs/date/isTomorrow.ts (hunk 2 above)
Comment (peterdemartini) [reply packages/data-mate/src/function-configs/date/isTomorrow.ts:35]: Haha we gave him the same comment and he just fixed it

**Review by ciorg — APPROVED** [review]

File: packages/data-mate/src/function-configs/date/isTomorrow.ts (hunk 2 above)
Comment (ciorg) [reply packages/data-mate/src/function-configs/date/isTomorrow.ts:35]: nice

**Pull request merged**